sidebars.js

```diff
@@ -123,155 +168,130 @@ module.exports = {
                 { type: 'doc', id: 'guides/connecting_to_your_data/cloud/azure/spark' }
               ]
             },
-
             {
               type: 'category',
-              label: '🍏 Core Skills',
+              label: '🔬 Advanced',
               items: [
-                {
-                  type: 'category',
-                  label: '✳️  How to get a Batch of data from a configured Datasource',
-                  items: [{ type: 'doc', id: 'guides/expectations/how-to-create-a-new-expectation-suite-using-the-cli' }]
-                },
-
-                {
-                  type: 'category',
-                  label: '✳️  How to create a Batch of data from a single file or in-memory dataframe',
-                  items: [{ type: 'doc', id: 'guides/expectations/how-to-create-a-new-expectation-suite-using-the-cli' }]
-                },
-
-                {
-                  type: 'category',
-                  label: '✳️  How to configure a DataConnector to introspect and partition tables in SQL',
-                  items: [{ type: 'doc', id: 'guides/expectations/how-to-create-a-new-expectation-suite-using-the-cli' }]
-                },
-                {
-                  type: 'category',
-                  label: '✳️  How to configure a DataConnector to introspect and partition a file system or blob store',
-                  items: [{ type: 'doc', id: 'guides/expectations/how-to-create-a-new-expectation-suite-using-the-cli' }]
-                }
-
+                'guides/connecting_to_your_data/advanced/database_credentials',
+                'guides/connecting_to_your_data/advanced/how-to-create-a-batch-from-a-sql-query',
+                'guides/connecting_to_your_data/advanced/how-to-create-a-lightweight-data-catalog-by-applying-a-descriptive-profiler-to-a-configured-datasource',
+                'guides/connecting_to_your_data/advanced/how-to-explore-changes-in-data-over-time-using-a-configured-datasource'
               ]
+            }
+          ]
+        },
+        {
+          type: 'category',
+          label: '🧪 Creating and editing Expectations for your data',
```

Comment:
This category seems to be missing all the core skills. Also, the order here does not match the spec: https://github.com/superconductive/design/blob/main/docs/20210421_knoxpod_golden_paths_mental_map/toc.md#creating-custom-expectations-the-article-in-this-section-need-examination-we-might-be-able-to-consolidate